.github/workflows/continuous-integration-workflow.yml

```diff
@@ -51,16 +51,12 @@ jobs:
           push: true
           tags: ${{ secrets.DOCKERHUB_USERNAME }}/${{ secrets.DOCKERHUB_REPOSITORY }}:latest
   job-three:
-    name: Deploy Heroku
+    name: Run Azure Login with OpenID Connect
     runs-on: ubuntu-latest
     needs: [job-one, job-two]
+    permissions:
+      id-token: write
+      contents: read
```

Comment:
Unless I'm mistaken, these permissions are [for accessing permission to make changes in GitHub](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#example-1-passing-the-github_token-as-an-input) and are unnecessary for the step making a call to Azure below